agent/engine/dockerstate/docker_task_engine_state.go

```diff
@@ -22,6 +22,8 @@ import (
 	apitask "github.com/aws/amazon-ecs-agent/agent/api/task"
 	"github.com/aws/amazon-ecs-agent/agent/engine/image"
 	apieni "github.com/aws/amazon-ecs-agent/ecs-agent/api/eni"
+	apiebs "github.com/aws/amazon-ecs-agent/ecs-agent/api/resource"
```

Comment:
nit: if this is a generic resource to be reused for new resource types in the future, then should we import as `apiresource`?

Reply:
Good call out